jinahub/indexers/dbms/FileDBMSIndexer/__init__.py

```diff
@@ -334,3 +334,23 @@ def physical_size(self) -> int:
         """
         root_directory = Path(self.workspace)
         return sum(f.stat().st_size for f in root_directory.glob('**/*') if f.is_file())
+
+    def _get(self, docs: DocumentArray, **kwargs):
+        """Get a document by its id
+
+        :param docs: the documents
+        """
+        self._search(docs, is_update=False)
+
+    def _search(self, docs: DocumentArray, is_update):
+        for i, doc in enumerate(docs):
+            vec, doc_bytes = pickle.loads(self._query([doc.id])[0])
+            serialized_doc = Document(doc_bytes)
```

Comment:
isnt there a way to pickle in batch?

Reply:
won't investigate more into FileDBMS, since it will be removed